src/Alarm.cpp

```diff
@@ -53,31 +53,31 @@ void Alarm::determineBasePhotoresistorReading(){
     _greenLED->flash();
   }
   _baseReading = avgReading / numOfReadings;
-  Logger::Log("Base Photoresistor Reading Determined");
+  Serial.write(24021); //24021 = Data, Info, Base Photoresistor Data
```

Comment:
This isn't quite right. In the notes of the [Message Translation Chart](https://github.com/Lastrellik/Calico-Home-Security/blob/4f540c026c61163b1bf5dc57533bb2c4d0915a3d/docs/development/logging.md#message-translation-chart) it points out that `Log Level` is only relevant for Log Type (1) messages. Since this is a Data message (2) then the `Log Level` number should be `0` for `N/A`